emctl/cmd/client/command/meshclient/zz_canary_gen.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright (c) 2017, MegaEase
```

Comment:
There is still warning in generated files about heading comments. I think we need more `*` 😄